src/lib/features/auth/specs/login-button.spec.ts

```diff
@@ -0,0 +1,11 @@
+import svelte from 'svelte-inline-compile';
+import { LoginButton } from '..';
+import { useLocalVars } from '$lib/shared/lib/jest-hacks';
+import { render } from '@testing-library/svelte';
+
+it('renders the button', async () => {
```

Comment:
issue: I think it's important for the user that the button is clickable and actually calls the authentication flow on press, which is not covered by the tests currently

Reply:
I would really like if you provide an idea on how to actually do it

Reply:
You need to mock the `$lib/features/authenticate` module, trigger a click on a button with `@testing-library/user-event` and then check the mock to see if it has been called or not